fe/fe-core/src/main/java/org/apache/doris/datasource/iceberg/IcebergMetadataCache.java

```diff
@@ -52,6 +53,7 @@ public class IcebergMetadataCache {
     private final LoadingCache<IcebergMetadataCacheKey, List<Snapshot>> snapshotListCache;
     private final LoadingCache<IcebergMetadataCacheKey, Table> tableCache;
     private final LoadingCache<IcebergMetadataCacheKey, IcebergSnapshotCacheValue> snapshotCache;
+    private final LoadingCache<IcebergMetadataCacheKey, View> viewCache;
```

Comment:
And comment to explain why need this cache and the difference between `viewCache` and `tableCache`